index.bs

```diff
@@ -3676,85 +3690,85 @@ the same procedure as other [=assertion signatures=] generated by the [=authenti
             </td>
         </tr>
         <tr>
             <td>`1`</td>
             <td>`1`</td>
             <td>
-                The credential is a multi-device credential
+                The credential is a [=multi-device credential=].
             </td>
         </tr>
     </table>
     <figcaption>
-        [=Authenticator data=] bits 3 and 4 combinations 
+        `BE` and `BS` [=flag=] combinations
     </figcaption>
 </figure>
 
-It is recommmended that [=Relying Party|Relying Parties=] store the last value of these bits with the user account for future evaluation.
+It is RECOMMENDED that [=[RPS]=] store the most recent value of these [=flags=] with the [=user account=] for future evaluation.
 
-The following is a non-normative, non-exhaustive list of how [=Relying Party|Relying Parties=] may utilize these bits:
+The following is a non-normative, non-exhaustive list of how [=[RPS]=] might utilize these [=flags=]:
 
     - Requiring additional [=authenticators=]:
 
-        When bit 3 is set to `0`, the [=authenticator=] will never allow the credential to transition from a single-device credential to
-        a multi-device credential. 
+        When `BE` [=flag=] is set to `0`, the [=generating authenticator=] will never allow the credential to transition from a 
+        [=single-device credential=] to a [=multi-device credential=]. 
         
-        A single-device credential is not durable and cannot survive single device loss. [=Relying Party|Relying Parties=]
-        should ensure that an account has additional [=authenticators=] enrolled and/or an account recovery process in place.
+        A [=single-device credential=] is not durable and cannot survive single device loss. [=[RPS]=] should ensure that a [=user account=] 
+        has additional [=authenticators=] [=registration ceremony|registered=] and/or an account recovery process in place.
 
-        For example, the user could be prompted to set up a single-device credential on a [=roaming authenticator=], like a 
-        security key, or another [=authenticator=] that is capable of holding multi-device credentials.
+        For example, the user could be prompted to set up an additional [=authenticator=], such as a [=roaming authenticator=] or an 
+        [=authenticator=] that is capable of [=multi-device credentials=].
     
     - Upgrading a user to a password-free account:
 
-        When bit 4 changes from `0` to `1`, the [=authenticator=] is signaling that the [=Public Key Credential Source|credential source=]
-        is durable (it has been backed up and is protected from single device loss). This is often referred to as a multi-device credential.
+        When the `BS` [=flag=] changes from `0` to `1`, the [=authenticator=] is signaling that the [=credential=] is backed up and is protected from single device loss. 
+        This is often referred to as a [=multi-device credential=]
```

Comment:
This is inconsistent with the Backup Eligibility definition: the credential is already a _multi-device credential_ even when the BS flag is still 0.